backend/tournesol/tests/test_api_unconnected_entities.py

```diff
@@ -187,9 +205,12 @@ def test_all_linked_should_return_empty(self):
         self.assertEqual(response.data["count"], 0)
 
 
-class SimpleNotAllConnectedTest(TestCase):
+class TwoIsolatedGraphsTestCase(TestCase):
```

Comment:
The righter term would be TwoIsolatedGraphComponentsTestCase

Reply:
There is always a single graph even if it has multiple unconnected components

Reply:
Thanks, I'll fix this.